jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/AbstractConnector.java

```diff
@@ -321,6 +323,37 @@ public boolean isShutdownDone()
         LOG.info("Started {}", this);
     }
 
+    public ComplianceViolation.Listener getComplianceViolationListener()
```

Comment:
Going to move this entire logic to HttpConfiguration instead.
Per ideas mentioned at https://github.com/jetty/jetty.project/pull/11301#discussion_r1462675117

Reply:
This should be documented in the adoc documentation as well.
This PR should include that documentation update.
The porting guide might need to be updated as well.